# Comments provide naming from Technical Customer Information

std_msgs/Header header
# SenStInfo_SenSt
uint8 sensor_state
